plugins/package-managers/pub/src/main/kotlin/model/Pubspec.kt

```diff
@@ -113,6 +114,9 @@ private object DependencyMapSerializer : KSerializer<Map<String, Dependency>> by
 
     private fun YamlNode.decodeDependency(): Dependency {
         if (this is YamlScalar) return HostedDependency(yamlScalar.content)
+        // https://dart.dev/tools/pub/dependencies says that "The version constraint is optional but recommended. If no
+        // version constraint is given, any is assumed.".
```

Comment:
Shouldn't this better go to documentation for the property in the class?

Reply:
I've made another PR: https://github.com/oss-review-toolkit/ort/pull/9313